.github/workflows/pr-title-check.yml

```diff
@@ -0,0 +1,22 @@
+name: PR Title Check
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  pr-title:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Validate PR Title
+        run: |
+          if [[ ! "${{ github.event.pull_request.title }}" =~ ^(feat|fix|chore|major)\([a-zA-Z0-9_-]+\):\ .+$ ]]; then
+            echo "O título do Pull Request deve seguir a convenção: 'tipo(contexto): descrição'."
+            echo "Os tipos devem ser: feat, fix, chore, major."
+            echo "Exemplo de título correto: feat(component): create link component"
```

Comment:
(block) the open source projets is always in english

Reply:
https://github.com/juntossomosmais/atomium/pull/526/commits/537e8a3595e437a67a3e1da0c409343e0712c143